### PR TITLE
Linetools IO for IGMGuesses

### PR DESCRIPTION
**Pull request by ntejos**
Some utilities for IO capabilities for IGMGuesses in pyigm

### CODE REVIEW

File: linetools/isgm/abscomponent.py [hunk 1]
```diff
@@ -243,15 +243,21 @@ def from_dict(cls, idict, coord=None, **kwargs):
...
+        # slf = cls(radec, tuple(idict['Zion']), idict['zcomp'], Quantity(idict['vlim'], unit='km/s'),
 
         # backwards compatibility
-        for key in ['reliability', 'Reliability']:
-            if key in idict.keys():
-                reliability = idict[key]
-                break
-            else:
-                reliability = 'none'
+        if 'reliability' in idict.keys():
```
Comment (profxj) [inline linetools/isgm/abscomponent.py]: I can't see how the old code doesn't do exactly the same thing.
But you must have a reason..

File: linetools/isgm/io.py [hunk 2]
```diff
@@ -179,3 +181,100 @@ def write_joebvp_from_components(comp_list, specfile, outfile):
         s = comp.repr_joebvp(specfile, flags=flags, b_default=b_val)  # still, b values from abslines take precedence if they exist
         f.write(s)
     f.close()
+
+
+def read_igmg_to_components(igmguesses_json):
```
Comment (profxj) [inline linetools/isgm/io.py]: IGMGuesses is a pyigm construct.
I think this method needs to go there.

And, I'll note that IGMSightline probably 
already does most of what you want here.

File: linetools/isgm/io.py [hunk 3]
```diff
@@ -179,3 +181,100 @@ def write_joebvp_from_components(comp_list, specfile, outfile):
...
+        comp_list += [comp]
+    # sort list by redshift
+    redshifts = [comp.zcomp for comp in comp_list]
+    inds = np.argsort(redshifts)
+    comp_sorted = list(np.array(comp_list)[inds])
+    #return
+    return comp_sorted
+
+
+def write_igmg_from_components(comp_list, specfile, fwhm, outfile='IGM_model.json'):
```
Comment (profxj) [inline linetools/isgm/io.py]: Again, this should be in pyigm.  And 
make it work with IGMSightline which
I don't think has a write() method yet.

**Comment by ntejos** [comment]
You are right, will close this PR and move the code to pyigm. Thanks.

File: linetools/isgm/abscomponent.py (hunk 1 above)
Comment (ntejos) [reply linetools/isgm/abscomponent.py]: I found it clearer in the new code. but should be doing the same.

**Comment by profxj** [comment]
merging.

**Pull request merged**